tutorials/plot_events_and_annotations.py

```diff
@@ -0,0 +1,120 @@
+"""
+The **events** and :class:`Annotations <mne.Annotations>` data structures
+=========================================================================
+
+Events and annotations are quite similar. This tutorial highlights their
+differences and similitudes and tries to shade some light to which one is
+preferred to use in different situations when using MNE.
+Here follows both terms definition from the glossary.
+
+    events
+        Events correspond to specific time points in raw data; e.g., triggers,
+        experimental condition events, etc. MNE represents events with integers
+        that are stored in numpy arrays of shape (n_events, 3). Such arrays are
+        classically obtained from a trigger channel, also referred to as stim
```

Comment:
stim is really an abbreviation. Since it's a tutorial, I'd suggest expanding to the full name

Reply:
My only goal here was just to replicate whatever is in glossary. Once glossary is linkable I would remove it.